docs/configuration/entrypoints.md

```diff
@@ -239,11 +239,13 @@ TLS Mutual Authentication can be `optional` or not.
 * If `optional = true`, if a certificate is provided, verifies if it is signed by a specified Certificate Authority (CA). Otherwise proceeds without any certificate.
 * If `optional = false`, Traefik will only accept clients that present a certificate signed by a specified Certificate Authority (CA).
 
+!!! warning
```

Comment:
You messed up the formatting, so the output does not look like a note (check it out in https://deploy-preview-5434--traefik-doc.netlify.com/configuration/entrypoints/).
I think you need a blank line after L242, and then you need to indent the whole paragraph. Have a look at the other warning note at the end of that file.